libs/ui-theme/src/lib/hooks/useScreen.ts

```diff
@@ -0,0 +1,12 @@
+import { Grid } from 'antd';
+
+const { useBreakpoint } = Grid;
+
+export const useScreen = () => {
```

Comment:
please lmk if you think the `libs/ui-theme` is not the correct library for this hook